modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -802,7 +813,7 @@ GrainTracker::trackGrains()
         // Must be a nucleating grain (status is still not set)
         if (grain._status == Status::CLEAR)
         {
-          auto new_index = getNextUniqueID();
+          auto new_index = 0; //getNextUniqueID();
```

Comment:
Probably a mistake, this needs to be changed back for nucleation test cases.

Reply:
Oh yes. I accidentally uploaded this.